deno2/.gn

```diff
@@ -8,7 +8,7 @@ buildconfig = "//build/config/BUILDCONFIG.gn"
 # The secondary source root is a parallel directory tree where
 # GN build files are placed when they can not be placed directly
 # in the source tree, e.g. for third party source trees.
-secondary_source = "//v8/"
+#secondary_source = "//v8/"
```

Comment:
Why is this commented out?

Reply:
Removed completely. Symlinks are introduced here that circumvent the need for this.